Website/index.js

```diff
@@ -0,0 +1,471 @@
+function show_personalised() {
+    $("#updated").text(jsonData["updated"]);
+    $("#date2").remove();
+
+    $("#personalised").empty();
+
+    var day = new Date(String($("#start").val()));
+    days_of_week(day);
+    var week;
+
+    function days_of_week(day) {
+        $(Object.keys(jsonData["weeks"])).each(function (i, v) {
+
+            $(jsonData["weeks"][v]).each(function (b, value) {
+                var a = new Date(value);
+
+                if (String(day) == String(a)) {
+                    week = jsonData["weeks"][v]
+                };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
                }
```